app/components/Button/Button.css

```diff
@@ -30,7 +30,7 @@
   }
 
   &:hover:not(:disabled) {
-    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+    box-shadow: 0 1px 2px rgba(0, 0, 0, 10%);
```

Comment:
I hate this rule change, but oh well :shrug:

Reply:
I kinda regret not changing the rule and sticking with the decimal format 🤔 I might change back, but not tonight ...